config/config.php

```diff
@@ -12,24 +12,15 @@
  * Obviously, if you use closures in your config you can't cache it.
  */
 
-$cachedConfigFile = 'data/cache/app_config.php';
+$configManager = new ConfigManager([
+    // Load config from all modules first
+    new PhpFileProvider('src/*/Config/{{,*.}global,{,*.}local}.php'),
```

Comment:
> But what is the best to do? Only allow global.php or just .php?

If you don't want to use provider classes, then just go for `*.php` within some directory - it's pretty straight forward.